src/test/java/catalinazoldi/tests/Tema9/Tema8/tests/BasicAjaxTest.java

```diff
@@ -0,0 +1,35 @@
+package catalinazoldi.tests.Tema9.Tema8.tests;
+
+
+import catalinazoldi.tests.Tema8.pageObjects.BasicAjaxPage;
```

Comment:
Replace this import with the one from Tema9 package -> `import catalinazoldi.tests.Tema9.Tema8.pageObjects.BasicAjaxPage;`